src/css/adjustCSS.js

```diff
@@ -0,0 +1,60 @@
+define([
+	"../core",
+	"../var/rfxnum"
+], function( jQuery, rfxnum ) {
+
+function adjustCSS( elem, prop, value, tween ) {
+	var adjusted,
+		scale = 1,
+		maxIterations = 20,
+		currentValue = tween ?
+			function() { return tween.cur(); } :
+			function() { return jQuery.css( elem, prop, "" ); },
+		initial = currentValue(),
+		unit = value && value[ 3 ] || ( jQuery.cssNumber[ prop ] ? "" : "px" ),
+		// Starting value computation is required for potential unit mismatches
+		recasted = ( jQuery.cssNumber[ prop ] || unit !== "px" && +initial ) &&
+			rfxnum.exec( jQuery.css( elem, prop ) );
```

Comment:
Seeing it all together, maybe a couple more renames are in order:
* `value` → `valueParts`
* `recasted` → `initialInUnit`